app/models/service.rb

```diff
@@ -15,7 +15,7 @@ class Service < ApplicationRecord
 
   has_system_name uniqueness_scope: :account_id
 
-  attr_readonly :system_name
+  attr_readonly :system_name, :admin_support_email, :tech_support_email
```

Comment:
No need of this, those attributes were removed

Reply:
@hallelujah At the beginning I thought the same, but later I understood why:
![image](https://user-images.githubusercontent.com/11318903/55472939-9cfe8400-560d-11e9-8c79-3f9c01ab23bb.png)

Reply:
Yes the usage is removed, but not the attribute so technically someone could update it